distributed/worker.py

```diff
@@ -1021,7 +1039,7 @@ def __init__(
         )
         self.periodic_callbacks["keep-alive"] = pc
 
-        pc = PeriodicCallback(self.find_missing, 1000)
+        pc = PeriodicCallback(self.find_missing, 1000)  # type: ignore
```

Comment:
Work around bug in tornado type annotations that declare that callback can only be a sync function, whereas we pass async functions (and they work fine).

Reply:
- https://github.com/tornadoweb/tornado/issues/3117